components/tooltip/index.css

```diff
@@ -122,6 +122,7 @@ governing permissions and limitations under the License.
   }
 
   &.is-open {
+    --spectrum-overlay-animation-distance: var(--mod-tooltip-animation-distance, var(--spectrum-tooltip-animation-distance));
```

Comment:
I'm a bit torn on this approach. Popover and Tooltip are the only components that import `"../commons/overlay-coretokens.css"`. And I'm overriding the animation distance value for each of them (popover changes happening in #2246). On one hand, it's nice to have the overlay transform in one place to keep the code DRY...on the other hand, if each use their own values and we have to override the custom property each time, is it really all that DRY? Definitely open to opinions and thoughts on this.

Reply:
Yeah, I'm also leaning towards the direction of just defining the transforms within tooltip and popover. That change will make things more readable as well.

QuickAction appears to be inheriting from `"../overlay/index.css"` so it should be unaffected.

Reply:
The changes to refactor this look great. Nice work.